src/main/java/com/hibernate/dao/ShoppingCardDao.java

```diff
@@ -0,0 +1,12 @@
+package com.hibernate.dao;
+
+import com.hibernate.model.ShoppingCard;
+import com.hibernate.model.User;
+
+public interface ShoppingCardDao {
```

Comment:
```suggestion
public interface ShoppingCartDao {
```
save everywhere - change `d` to `t` please